modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/util/StreamInterceptorsLoader.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we change the licence year?

Reply:
:D Consequences of coding at the end of the year...

Reply:
updated : p